libraries/AP_Scripting/AP_Scripting.cpp

```diff
@@ -96,6 +96,16 @@ void AP_Scripting::init(void) {
         return;
     }
 
+    const char *dir_name = SCRIPTING_DIRECTORY;
+
+    if (!AP::FS().mkdir(dir_name)) {
+        gcs().send_text(MAV_SEVERITY_INFO, "Lua: created new directory (%s)", dir_name);
+    }
+    else if (errno != EEXIST ) {
```

Comment:
hum ... that a shame that we get difference between the backend ... on FATFS we don't need to zero the errno, but on posix we need to ... 

And the else if should be in the same line as the closing }

Reply:
@khancyr thanks for the review! I'll fix the formatting issue, but I don't understand what you're saying about ERRNO and posix..... Can you please clarify if possible?

Reply:
we should fix the formatting on this line.  So it should be "} else .. " instead of being split over two lines

Reply:
Formatting issue has been fixed. Thanks for the review :)